portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -70,10 +70,12 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       String commentText = (String) entity.getProperty("commentText");
       String commentAuthor = (String) entity.getProperty("commentAuthor");
       String imageUrl = (String) entity.getProperty("attachedImage");
+      String showEmail = (String) entity.getProperty("showEmail");
```

Comment:
The value of showEmail will be passed back to the client in a JSON string, so eventually it will have to be converted back into a string. In this case, should it still be made into a boolean first?

Reply:
Even if JSON is sent as a string, JSON can still represent boolean types, so it should be a boolean. Ideally, you'd store it in the datastore as a boolean, and also store it in `comment` as a boolean (the easiest way to do that would be to change `comment` into a `HashMap<String, Object>`). This way, you can always treat `showEmail` as a boolean, even in the javascript code.

Reply:
Yep okay, that makes sense! Thanks for the feedback and tips 😊